### ISSUE TITLE
Missing info about how to apply licenses

### ISSUE THREAD

**Issue by duartemtorres** [github/choosealicense.com#1263]
Each license page has a section named "How to apply this license" typically saying:

> Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. Replace [year] with the current year and [fullname] with the name (or names) of the copyright holders.

Some licenses like the [MIT License](https://choosealicense.com/licenses/mit/) include a step to replace the copyright templates with your own identifying information. Something like:

> Replace [year] with the current year and [fullname] with the name (or names) of the copyright holders.

However, the [Apache License 2.0](https://choosealicense.com/licenses/apache-2.0/) is missing this. I do not know if any other license has the same issue but it would be useful to have this to guide newbies.

**Comment by josiahaltschuler** [comment]
You are correct, @duartemtorres. Toward the bottom of the Apache 2.0 license there is this line:
> Copyright [yyyy] [name of copyright owner]

So in the "How to apply this license" section of that page maybe the added line could be:
> Replace [yyyy] with the current year and [name of copyright owner] with the name (or names) of the copyright holders.

@mlinksva, I'd be happy to add it in a PR if you want. Just let me know.

**Comment by mlinksva** [comment]
This is not correct. The LICENSE file in the case of Apache-2.0 and other licenses with deployment instructions at the end should not be modified. The copyright info should be added to a source file comment when following the deployment instructions

There's a link to the Apache Software Foundation's own guidance on our page. You can see that they do not modify that text in the LICENSE file itself in various ASF projects.

**Comment by josiahaltschuler** [comment]
Hi @mlinksva, thanks for the explanation. I did not realize that. I see the copyright boilerplate now on various ASF projects as you mentioned. Sorry for the confusion.
